### PR TITLE
Fix the build error and custom background deletion bug

### PR DESCRIPTION
**Pull request by Gauravjeetsingh**
Currently, it was resetting the background even if we delete an inactive custom background, fixed that.

Also, resolved linting errors and remove unused code.

### CODE REVIEW

File: www/main/settings/components/ThemeContainer.jsx [hunk 1]
```diff
@@ -81,8 +81,12 @@ const ThemeContainer = () => {
                 onRemove={() => {
                   removeCustomBackgroundFile(tile['background-image-path'].replace(/\\(\s)/g, ' '));
                   upsertCustomBackgrounds(setCustomThemes);
-                  const currentThemeInstance = themes.filter((theme) => theme.key === currentTheme);
-                  setDefaultBg(currentThemeInstance, setThemeBg, themeBg);
+                  if (tile['background-image'] === themeBg.url.href) {
+                    const currentThemeInstance = themes.filter(
+                      (theme) => theme.key === currentTheme,
+                    );
+                    setDefaultBg(currentThemeInstance, setThemeBg, themeBg);
```
Comment (theamanjs) [inline www/main/settings/components/ThemeContainer.jsx:88]: The scope of `currentThemeInstance` variable is inside the if statement only and it is being used outside of that if statement which is a violation. Veerji this should be fixed IMO

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: www/main/settings/components/ThemeContainer.jsx (hunk 1 above)
Comment (Gauravjeetsingh) [reply www/main/settings/components/ThemeContainer.jsx:88]: It is inside if statement, if statement, ends at line 89.

Comment (theamanjs) [reply www/main/settings/components/ThemeContainer.jsx:88]: I messed up veerji @Gauravjeetsingh really sorry for this 🙏

**Review by theamanjs — APPROVED** [review]

**Comment by theamanjs** [comment]
@inderpreetsingh veerji this PR is ready to be merged after your review.

**Review by inderpreetsingh — APPROVED** [review]

**Pull request merged**